app/graphql/types/wallets/create_input.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Types
+  module Wallets
+    class CreateInput < Types::BaseInputObject
+      description 'Create Wallet Input'
+
+      argument :currency, Types::CurrencyEnum, required: true
+      argument :customer_id, ID, required: true
+      argument :expiration_at, GraphQL::Types::ISO8601DateTime, required: false
+      argument :granted_credits, String, required: true
+      argument :name, String, required: false
+      argument :paid_credits, String, required: true
+      argument :rate_amount, String, required: true
```

Comment:
Should we use `Float` rather than `String` for these arguments?

Reply:
Hmm, we have `::Validators::DecimalAmountService` that is used for wallet attributes which are handled as BigDecimal and we accept only strings to avoid bad parsing. This particular attribute has not been added with this feature (I just needed to move it to special input class based on linter warnings) but basically the same idea was used for attributes that I had added...

Reply:
The same rule for these big decimal attribute is also applied on the API side...